packages/nextjs/pages/events.tsx

```diff
@@ -10,7 +10,7 @@ const Events: NextPage = () => {
   const { data: buyTokenEvents, isLoading: isBuyEventsLoading } = useScaffoldEventHistory({
     contractName: "Vendor",
     eventName: "BuyTokens",
-    fromBlock: 0,
+    fromBlock: 0n,
   });
 
   // // SellTokens Events
```

Comment:
I think we need to change the commented `// fromBlock: 0,` for `0n` in SellToken Events also. We'll save some time to the people wondering why their Sell Events are not showing 🙃

Reply:
thanks, updated